src/components/QuizResult.vue

```diff
@@ -37,18 +37,12 @@ export default {
       type: Object,
       required: true,
     },
-    goToStart: {
-      type: Function,
-      required: true,
-    },
   },
 
-  computed: {
-    imageBoxStyle() {
-      return {
-        backgroundImage: `url(${this.result.img || DEFAULT_IMAGE_URL})`
-      };
-    },
+  data() {
+    return {
+      defaultImageUrl: DEFAULT_IMAGE_URL,
```

Comment:
Instead of data property you could easily use computed property and provide it as a fallback there. WDYT?

Reply:
definitely much better 👍